Services/Common/OJS.Services.Common/Implementations/ContestsActivityService.cs

```diff
@@ -74,17 +83,38 @@ public bool CanBeCompeted(IContestForActivityServiceModel contest)
             return contest.StartTime <= now;
         }
 
-        return contest.StartTime <= now && now <= contest.EndTime;
+        //If the above conditional statements are not entered, first we have to check
+        //the the start and end time > check if start is before now and end is after now
+        //if this check returns false we have to check if the current user is a participant
+        //with a remaining time in an online contest
+        return (contest.StartTime <= now && now <= contest.EndTime) ||
```

Comment:
"now" seems kind of an odd name for a variable. Can you try to rename it to a better one?

Reply:
Renaming it to currentTimeInUtc